examples/gripper_controller.py

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env python3
+
+# A basic example of using BlueInterface for gripper control.
+# It allows a user to open and close the gripper.
+
+import sys
+from blue_interface import BlueInterface
+import numpy as np
+
+side = "right"
+ip = "127.0.0.1"
+blue = BlueInterface(side, ip)
+
+blue.calibrate_gripper()
+
+opened = True
```

Comment:
With the new calibrate gripper service, the gripper will be closed after calibration. We should probably that behavior in ```blue_core```.